### PR TITLE
SWC-6636: Replace GWT Certification Quiz implementation with React implementation

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/view/SettingsViewImpl.java [hunk 1]
```diff
@@ -269,11 +269,7 @@ public SettingsViewImpl(
       presenter.goTo(new Quiz("Certification"))
     );
     certificationPassedButton.addClickHandler(event ->
-      DisplayUtils.newWindow(
-        "https://help.synapse.org/docs/User-Types.2007072795.html#UserAccountTiers-CertifiedUsers",
-        "_blank",
-        ""
-      )
+      presenter.goTo(new Quiz("Certification"))
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/view/SettingsViewImpl.java:272]: An unrelated fix for an issue that Xa noticed.  The "Certification Passed" button used to go to the documentation instead of showing the passing record.  Note that the documentation is still available from the More Info button below this one.

File: src/main/java/org/sagebionetworks/web/client/presenter/QuizPresenter.java [hunk 2]
```diff
@@ -1,379 +1,33 @@
 package org.sagebionetworks.web.client.presenter;
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/presenter/QuizPresenter.java:2]: A shell of a class that it used to be.  Now it just loads the SRC implementation.

**Comment by jay-hodgson** [comment]
Not in experimental mode!

Screen Shot 2024-02-06 at 11 15 00 AM

Screen Shot 2024-02-06 at 11 16 58 AM

**Review by nickgros — APPROVED** [review]
